README.md

```diff
@@ -61,38 +62,44 @@ Importing a one-time snapshot from the registry
 
 You can download a dump of all packages and import it into your local registry for development:
 
-```
+```console
 curl https://code.dlang.org/api/packages/dump > mirror.json
 dub -- --mirror=mirror.json
 ```
 
 Starting the registry with `mirror.json` will import all packages within the JSON file.
 Once all packages have been imported, you can start the registry as you normally would:
 
-```
+```console
 dub
 ```
 
 And you should notice that it now contains all packages which are listed on code.dlang.org
 
 Note that `--mirror=mirror.json` and `--mirror=https://code.dlang.org` are very similar and the `mirror.json` is only preferred for local development because it allows to easily nuke the entire mongo database and re-initialize it without needing any connection to the internet.
 
-Deploy your private dub-registry with Docker
---------------------------------------------
+Build/Running dub-registry with Docker
+---------------------------------------
 
-The [dlangcommunity/dub-registry](https://hub.docker.com/r/dlangcommunity/dub-registry/) Docker image is available for an easy setup:
+- Build:
 
-```
-export DUB_REGISTRY_HOME="$PWD"
-docker run --rm -ti -p 9095:9095 -v $DUB_REGISTRY_HOME:/bitnami -v $DUB_REGISTRY_HOME:/dub dlangcommunity/dub-registry
+```console
+export DUB_REGISTRY_HOME=$PWD
+docker compose build
 ```
 
-This will run both `mongodb` and `dub-registry` while persisting the database in the `$DUB_REGISTRY_HOME` location. The registry is accessible at http://127.0.0.1:9095
-
-To run it as a daemon and make it auto-restart use:
+- Run:
 
+```console
+docker compose up -d --remove-orphans
```

Comment:
`--remove-orphans` should not always be needed / does not need to be included in README.